subscriptions.go

```diff
@@ -167,6 +167,7 @@ type NewSubscription struct {
 	CustomerNotes           string               `xml:"customer_notes,omitempty"`
 	VATReverseChargeNotes   string               `xml:"vat_reverse_charge_notes,omitempty"`
 	BankAccountAuthorizedAt NullTime             `xml:"bank_account_authorized_at,omitempty"`
+	GatewayCode             string               `xml:"gateway_code,omitempty"`
```

Comment:
👍 

Found this here:
https://dev.recurly.com/docs/create-subscription

Reply:
> I need some help seeing where these fields are in Recurly's documentation. I wasn't able to match them up.
> 
> I believe we also need to include `GatewayCode` on the `Subscription` struct so it can be read back on an API call reading the subscription.

actually we can't include it in `Subscription` struct, as Recurly does not return it in subscription response. This is pretty weird field that you can only set it but not get it